packages/blocks/src/root-block/edgeless/components/auto-complete/edgeless-auto-complete.ts

```diff
@@ -258,10 +264,23 @@ export class EdgelessAutoComplete extends WithDisposable(LitElement) {
 
   private _addConnector(source: Connection, target: Connection) {
     const { service } = this.edgeless;
+
+    let strokeColor = '--affine-palette-line-grey';
```

Comment:
`--affine-palette-line-grey`  `--affine-tag-gray`

Can we standardize the naming of these?